From f4713b1094031d7c71b69579d555c0537280f5c5 Mon Sep 17 00:00:00 2001
From: Josef Stribny <jstribny@redhat.com>
Date: Tue, 27 May 2014 12:20:35 +0200
Subject: [PATCH 1/2] Rename option --files-are-digests to --file-is-digest

---

diff -Naur obs-sign-2.6.1.orig/signd obs-sign-2.6.1/signd
--- obs-sign-2.6.1.orig/signd	2022-07-29 12:59:54.000000000 +0200
+++ obs-sign-2.6.1/signd	2022-11-14 09:16:15.416146448 +0100
@@ -880,7 +880,7 @@
 }
 
 sub have_files_are_digests {
-  my ($status) = rungpg('/dev/null', undef, $gpg, '--files-are-digests', '--version');
+  my ($status) = rungpg('/dev/null', undef, $gpg, '--file-is-digests', '--version');
   return !$status;
 }
 
@@ -1233,7 +1233,7 @@
     } else {
       push @args, '-u', $user;
     }
-    return rungpg($phrasesfile, undef, $gpg, "--batch", "--force-v3-sigs", "--files-are-digests", "--digest-algo=$hashalgo", "--no-verbose", "--no-armor", "--no-secmem-warning", "--ignore-time-conflict", @pinentrymode, "--passphrase-fd=0", @args, "-sbo", "-", $hash);
+    return rungpg($phrasesfile, undef, $gpg, "--batch", "--force-v3-sigs", "--file-is-digests", "--digest-algo=$hashalgo", "--no-verbose", "--no-armor", "--no-secmem-warning", "--ignore-time-conflict", @pinentrymode, "--passphrase-fd=0", @args, "-sbo", "-", $hash);
   }
 }
 
diff -Naur obs-sign-2.6.1.orig/signd.8 obs-sign-2.6.1/signd.8
--- obs-sign-2.6.1.orig/signd.8	2022-07-29 12:59:54.000000000 +0200
+++ obs-sign-2.6.1/signd.8	2022-11-14 09:16:33.118233765 +0100
@@ -14,7 +14,7 @@
 
 signd uses the same configuration used for sign, /etc/sign.conf.
 It needs a gpg implementation that understands the
-"--files-are-digests" option to work correctly.
+"--file-is-digests" option to work correctly.
 
 .SH SECURITY
 Unless the allow-unprivileged-ports option is set to true in
