diff -Naur nanoblogger-3.4.2/plugins/entry/format/markdown.sh nanoblogger-3.4.2-new/plugins/entry/format/markdown.sh
--- nanoblogger-3.4.2/plugins/entry/format/markdown.sh	2010-02-14 21:42:17.000000000 +0100
+++ nanoblogger-3.4.2-new/plugins/entry/format/markdown.sh	2019-10-15 15:07:09.335282873 +0200
@@ -5,7 +5,7 @@
 : ${MARKDOWN_CMD:=markdown}
 : ${MARKDOWN_OPTS:= }
 
-nb_eval "$MARKDOWN_CMD" && MARKDOWN_INPATH=true
+nb_eval "$MARKDOWN_CMD --version" && MARKDOWN_INPATH=true
 	
 if [ "$MARKDOWN_INPATH" = "true" ]; then
 	# nb_msg "$plugins_entryfilteraction `basename $nb_plugin` ..."
diff -Naur nanoblogger-3.4.2/plugins/page/format/markdown.sh nanoblogger-3.4.2-new/plugins/page/format/markdown.sh
--- nanoblogger-3.4.2/plugins/page/format/markdown.sh	2009-08-10 02:01:37.000000000 +0200
+++ nanoblogger-3.4.2-new/plugins/page/format/markdown.sh	2019-10-15 15:07:30.190844997 +0200
@@ -5,7 +5,7 @@
 : ${MARKDOWN_CMD:=markdown}
 : ${MARKDOWN_OPTS:= }
 
-nb_eval "$MARKDOWN_CMD" && MARKDOWN_INPATH=true
+nb_eval "$MARKDOWN_CMD --version" && MARKDOWN_INPATH=true
 
 if [ "$MARKDOWN_INPATH" = "true" ]; then
 	nb_msg "$plugins_entryfilteraction `basename $nb_plugin` ..."
