From 563506fbe34142b2d4729895f190aeb6c6205eb2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Miroslav=20Such=C3=BD?= <msuchy@redhat.com>
Date: Thu, 6 Nov 2014 14:35:40 +0100
Subject: [PATCH] set default settings_dir

---
 pgtune | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pgtune b/pgtune
index 3d6996e..822af3c 100755
--- a/pgtune
+++ b/pgtune
@@ -610,7 +610,7 @@ def read_options(program_args):
     parser.add_option('-D', '--debug', action="store_true", dest="debug",
                       default="False", help="Enable debugging mode")
   
-    parser.add_option('-S', '--settings', dest="settings_dir", default=None, 
+    parser.add_option('-S', '--settings', dest="settings_dir", default="/usr/share/pgtune", 
                       help="Directory where settings data files are located at.  Defaults to the directory where the script is being run from")
 
     parser.add_option('--doctest', help='run doctests', action='store_true')
-- 
2.1.0

