diff -Naur marshmallow-3.11.1.orig/docs/conf.py marshmallow-3.11.1/docs/conf.py
--- marshmallow-3.11.1.orig/docs/conf.py	2021-03-30 19:51:32.829382094 +0200
+++ marshmallow-3.11.1/docs/conf.py	2021-03-30 19:52:00.738398342 +0200
@@ -14,7 +14,6 @@
     "sphinx.ext.viewcode",
     "alabaster",
     "sphinx_issues",
-    "autodocsumm",
 ]
 
 primary_domain = "py"
diff -Naur marshmallow-3.11.1.orig/setup.py marshmallow-3.11.1/setup.py
--- marshmallow-3.11.1.orig/setup.py	2021-03-30 02:11:23.000000000 +0200
+++ marshmallow-3.11.1/setup.py	2021-03-30 19:52:09.434403403 +0200
@@ -14,7 +14,6 @@
         "sphinx-issues==1.2.0",
         "alabaster==0.7.12",
         "sphinx-version-warning==1.1.2",
-        "autodocsumm==0.2.6",
     ],
 }
 EXTRAS_REQUIRE["dev"] = EXTRAS_REQUIRE["tests"] + EXTRAS_REQUIRE["lint"] + ["tox"]
