diff -Naur marshmallow-3.19.0/docs/conf.py marshmallow-3.19.0.orig/docs/conf.py
--- marshmallow-3.19.0/docs/conf.py	2022-11-30 14:46:21.904301634 +0100
+++ marshmallow-3.19.0.orig/docs/conf.py	2022-11-30 14:47:07.568642134 +0100
@@ -14,7 +14,6 @@
     "sphinx.ext.viewcode",
     "alabaster",
     "sphinx_issues",
-    "autodocsumm",
 ]
 
 primary_domain = "py"
diff -Naur marshmallow-3.19.0/setup.py marshmallow-3.19.0.orig/setup.py
--- marshmallow-3.19.0/setup.py	2022-11-30 14:46:21.904301634 +0100
+++ marshmallow-3.19.0.orig/setup.py	2022-11-30 14:46:37.181415549 +0100
@@ -14,7 +14,6 @@
         "sphinx-issues==3.0.1",
         "alabaster==0.7.12",
         "sphinx-version-warning==1.1.2",
-        "autodocsumm==0.2.9",
     ],
 }
 EXTRAS_REQUIRE["dev"] = EXTRAS_REQUIRE["tests"] + EXTRAS_REQUIRE["lint"] + ["tox"]
