diff -Naur marshmallow-3.17.0.orig/setup.py marshmallow-3.17.0/setup.py
--- marshmallow-3.17.0.orig/setup.py	2022-06-26 23:46:49.000000000 +0200
+++ marshmallow-3.17.0/setup.py	2022-06-27 19:07:26.976431607 +0200
@@ -14,7 +14,6 @@
         "sphinx-issues==3.0.1",
         "alabaster==0.7.12",
         "sphinx-version-warning==1.1.2",
-        "autodocsumm==0.2.8",
     ],
 }
 EXTRAS_REQUIRE["dev"] = EXTRAS_REQUIRE["tests"] + EXTRAS_REQUIRE["lint"] + ["tox"]
diff -Naur marshmallow-3.17.0.orig/docs/conf.py marshmallow-3.17.0/docs/conf.py
--- marshmallow-3.17.0.orig/docs/conf.py     2022-06-26 23:46:49.000000000 +0200
+++ marshmallow-3.17.0/docs/conf.py     2022-06-27 19:07:26.976431607 +0200
@@ -14,7 +14,6 @@
     "sphinx.ext.viewcode",
     "alabaster",
     "sphinx_issues",
-    "autodocsumm",
 ]
 
 primary_domain = "py"
