diff -Naur marshmallow-3.21.0.orig/docs/conf.py marshmallow-3.21.0/docs/conf.py
--- marshmallow-3.21.0.orig/docs/conf.py	2024-02-26 20:43:44.000000000 +0100
+++ marshmallow-3.21.0/docs/conf.py	2024-02-27 22:04:26.379014742 +0100
@@ -9,7 +9,6 @@
     "alabaster",
     "sphinx_issues",
     "versionwarning.extension",
-    "autodocsumm",
 ]
 
 primary_domain = "py"
diff -Naur marshmallow-3.21.0.orig/pyproject.toml marshmallow-3.21.0/pyproject.toml
--- marshmallow-3.21.0.orig/pyproject.toml	2024-02-26 20:43:44.000000000 +0100
+++ marshmallow-3.21.0/pyproject.toml	2024-02-27 22:05:53.435787582 +0100
@@ -37,7 +37,6 @@
   "sphinx-issues==4.0.0",
   "alabaster==0.7.16",
   "sphinx-version-warning==1.1.2",
-  "autodocsumm==0.2.12",
 ]
 tests = ["pytest", "pytz", "simplejson"]
 dev = ["marshmallow[tests]", "tox", "pre-commit~=3.5"]
