diff -Naur marshmallow-3.5.1.orig/src/marshmallow/schema.py marshmallow-3.5.1/src/marshmallow/schema.py
--- marshmallow-3.5.1.orig/src/marshmallow/schema.py	2020-03-05 14:01:01.000000000 +0100
+++ marshmallow-3.5.1/src/marshmallow/schema.py	2020-03-24 12:40:24.617005030 +0100
@@ -14,7 +14,7 @@
 from marshmallow import base, fields as ma_fields, class_registry, types
 from marshmallow.error_store import ErrorStore
 from marshmallow.exceptions import ValidationError, StringNotCollectionError
-from marshmallow.orderedset import OrderedSet
+from ordered_set import OrderedSet
 from marshmallow.decorators import (
     POST_DUMP,
     POST_LOAD,
