--- marshmallow/schema.py.orig	2015-09-07 11:56:10.277293096 +0200
+++ marshmallow/schema.py	2015-09-07 11:56:35.028387287 +0200
@@ -17,7 +17,7 @@
 from marshmallow import base, fields, utils, class_registry, marshalling
 from marshmallow.compat import (with_metaclass, iteritems, text_type,
                                 binary_type, OrderedDict)
-from marshmallow.orderedset import OrderedSet
+from ordered_set import OrderedSet
 from marshmallow.decorators import (PRE_DUMP, POST_DUMP, PRE_LOAD, POST_LOAD,
                                     VALIDATES, VALIDATES_SCHEMA)
 
