diff -Naur marshmallow-3.8.0.orig/src/marshmallow/schema.py marshmallow-3.8.0/src/marshmallow/schema.py
--- marshmallow-3.8.0.orig/src/marshmallow/schema.py	2020-09-16 11:21:17.000000000 +0200
+++ marshmallow-3.8.0/src/marshmallow/schema.py	2020-09-16 12:40:53.208395252 +0200
@@ -14,7 +14,7 @@
 from marshmallow import base, fields as ma_fields, class_registry, types
 from marshmallow.error_store import ErrorStore
 from marshmallow.exceptions import ValidationError, StringNotCollectionError
-from marshmallow.orderedset import OrderedSet
+from ordered_set import OrderedSet
 from marshmallow.decorators import (
     POST_DUMP,
     POST_LOAD,
