diff -Naur marshmallow-3.8.0.orig/docs/conf.py marshmallow-3.8.0/docs/conf.py
--- marshmallow-3.8.0.orig/docs/conf.py	2020-09-16 11:21:17.000000000 +0200
+++ marshmallow-3.8.0/docs/conf.py	2020-09-16 14:32:51.570352158 +0200
@@ -14,7 +14,6 @@
     "sphinx.ext.viewcode",
     "alabaster",
     "sphinx_issues",
-    "versionwarning.extension",
     "autodocsumm",
 ]
 
@@ -90,31 +89,3 @@
         "searchbox.html",
     ],
 }
-
-# sphinx-version-warning config
-versionwarning_messages = {
-    "latest": (
-        "This document is for the development version. "
-        'For the stable version documentation, see <a href="/en/stable/">here</a>.'
-    ),
-    "stable": (
-        "<strong>Black Lives Matter.</strong> "
-        'Support the <a href="https://eji.org/">Equal Justice Initiative</a> and '
-        '<a href="https://colorofchange.org/">Color of Change</a>.'
-    ),
-    "2.x-line": (
-        "marshmallow 2 is no longer supported as of 2020-08-18. "
-        '<a href="https://marshmallow.readthedocs.io/en/latest/upgrading.html#upgrading-to-3-0">'
-        "Update your code to use marshmallow 3</a>."
-    ),
-}
-# Show warning at top of page
-versionwarning_body_selector = "div.document"
-versionwarning_banner_title = ""
-# For debugging locally
-# versionwarning_project_version = "stable"
-
-
-def setup(app):
-    # https://docs.readthedocs.io/en/latest/guides/adding-custom-css.html
-    app.add_stylesheet("css/versionwarning.css")
